data/strings/strings.txt

```diff
@@ -29913,6 +30590,7 @@
     it = Questa applicazione richiede l'accesso alla tua posizione per la navigazione.
     ja = このアプリケーションは、ナビゲーションのために位置情報へのアクセスを必要とする。
     ko = 이 애플리케이션을 사용하려면 내비게이션 목적으로 사용자 위치에 액세스해야 합니다.
+    lv = Lai darbotos navigācija, lietotnei ir nepieciešama atrašanās vietas informācija.
```

Comment:
"Lai darbotos navigācija, lietotnei ir nepieciešama **ierīces** atrašanās vieta."

Reply:
I think "ierīces atrašanās vieta" sounds confusing (at least for me), as a device always have a location. Or is there any other reason to change it (length etc.)?

Or maybe you meant "**Lai darbotos navigācija, lietotnei ir nepieciešama ierīces atrašanās vietas informācija.**" to which I can agree.